.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish Python distributions
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sendgrid-python
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install build
```

Comment:
why not `make install` here?

Reply:
We don't need the virtual environment so I opted for a `pip install build` instead of the make install.